dom.bs

```diff
@@ -1270,7 +1270,7 @@ for discussion).
 <ol>
  <li><p>If <var>event</var>'s <a>stop propagation flag</a> is set, then return.
 
- <li><p>Let <var>listeners</var> be a new <a>list</a>.
+ <li><p>Let <var>listeners</var> be a new <a for=/>list</a>.
```

Comment:
(The problem with link defaults is that what we consider a shared ecosystem is in fact using contradictory terms after all. So ideally one of the sides in the dispute changes somehow.)